### DISCUSSION TITLE
How can they make updates so fast

### DISCUSSION THREAD

**Discussion by Cuteistfox** [CatimaLoyalty/Android#1327] [Q&A] [answered]
how are They made as fast like a machine?

**Comment by TheLastProject** [comment, marked as answer]
After multiple weeks of coding and lot of testing I released 2.23.0. Then I found out I made a mistake, but it was very easy to fix, so I quickly released 2.23.1 to fix it.

Then I had a long weekend because Monday was a Dutch public holiday, so I didn't have to go to work on Monday, so I also coded a bit extra for fun. And then someone emailed me that they found a pretty critical bug in 2.23.1, but it was very easy to fix, so I quickly fixed it.

2.23.1 and 2.23.2 are pretty small updates and I had a long weekend because of holidays, that's why I could do them so quickly :)

Hope that answers your question :)
